JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -593,12 +618,15 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   
   # mean estimates
   if(is.null(RE_estimates[["RE_mean"]]) && options[["RE_estimates"]]){
-    RE_estimates_mean <- createJaspTable(title = gettextf("Mean Estimates (%s)", "\u03BC"))
+    RE_estimates_mean <- createJaspTable(title = gettextf(
+      "Mean Estimates (%s)",
+      ifelse(options[["measures"]] == "correlation", "\u03C1" , "\u03BC")
```

Comment:
at some places like this you use `ifelse()` on another you do plain simple `if() ... else ...`. I think you should prefer the latter here and in use cases like that, right?

Reply:
ifelse is vectorised and can behave oddly. Especially in this case you'd like to use the standard if else statement.

Reply:
yes, changed